memgpt/cli/cli.py

```diff
@@ -100,9 +100,9 @@ def run(
     sys.stdout = original_stdout
 
     # overwrite the context_window if specified
-    if context_window is not None and int(context_window) != config.context_window:
+    if context_window is not None and int(context_window) != int(config.context_window):
         typer.secho(f"Warning: Overriding existing context window {config.context_window} with {context_window}", fg=typer.colors.YELLOW)
-        config.context_window = context_window
+        config.context_window = str(context_window)
```

Comment:
in this case, "reverse" cast to follow our assumption that `config.context_window` is stored as a str